examples/lightrag_api_oracle_demo.py

```
@@ -167,17 +184,35 @@ async def lifespan(app: FastAPI):
-                mode=request.mode, only_need_context=request.only_need_context
-            ),
-        )
-        return Response(status="success", data=result)
-    except Exception as e:
-        raise HTTPException(status_code=500, detail=str(e))
+    # try:
+    # loop = asyncio.get_event_loop()
+    if request.mode == "naive":
+        top_k = 3
+    else:
+        top_k = 60
```

Ruff (0.8.0)

Use ternary operator `top_k = 3 if request.mode == "naive" else 60` instead of `if`-`else`-block

Replace `if`-`else`-block with `top_k = 3 if request.mode == "naive" else 60`

(SIM108)